### DISCUSSION TITLE
Feature suggestion: New Expansion Pak tier (Enhanced)

### DISCUSSION THREAD

**Discussion by PW5190** [Polprzewodnikowy/N64FlashcartMenu#157] [Ideas]
We know the N64 Expansion Pak in games that have three tiers in the menu:

`Not required` = Used for most of the library.
`Recommended` = Used for Perfect Dark (NA/PAL).
`Required` = Used for DK64, Majora's Mask, and Perfect Dark (JP).

On the N64Brew Discord server, I discussed about a fourth tier in the menu, `Enhanced`. [Here's a list of games that use this tier.](https://nintendo.fandom.com/wiki/Nintendo_64_Expansion_Pak)

**Comment by networkfusion** [comment]
IMHO, that falls under "recommended"
